html/infrastructure/safe-passing-of-structured-data/shared-array-buffers/resources/iframe-domain-failure.sub.html

```diff
@@ -6,5 +6,6 @@
 "use strict";
 
 document.domain = "{{host}}";
+parent.postMessage({name: 'domain', value: document.domain}, "*");
```

Comment:
Perhaps also post crossOriginIsolated here so we can assert that's true?

Reply:
Done. The value is actually false because it is a cross-origin iframe. Added some comments in the other file.